run/browsers_test.py

```diff
@@ -28,28 +28,37 @@
     'os_version': None,
 }
 
+
 class TestBrowsers(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        filepath = os.path.join(os.path.dirname(__file__), os.pardir, 'browsers.json')
+        filepath = os.path.join(os.path.dirname(__file__),
+                                os.pardir, 'browsers.json')
         with open(filepath) as f:
             cls.browsers = json.load(f)
 
     def test_all_platforms_have_only_valid_characters(self):
         for platform_id, platform_info in self.browsers.items():
-            self.assertTrue(re.match(VALID_PLATFORM_ID_REGEX, platform_id),
-                'platform_id with invalid characters: %s' % platform_id)
+            self.assertTrue(
+                re.match(VALID_PLATFORM_ID_REGEX, platform_id),
+                'platform_id with invalid characters: %s' % platform_id
+            )
 
     def test_all_browsers_have_required_fields(self):
         for platform_id, platform_info in self.browsers.items():
             for key, valid_values in REQUIRED_PLATFORM_FIELDS.items():
-                self.assertTrue(key in platform_info.keys(),
-                    'Required field missing: %s (platform %s)' % (key, platform_id))
+                self.assertTrue(
+                    key in platform_info.keys(),
+                    'Required field missing: %s (platform %s)' %
+                    (key, platform_id)
+                )
 
                 if REQUIRED_PLATFORM_FIELDS[key] is not None:
-                    self.assertTrue(platform_info[key] in REQUIRED_PLATFORM_FIELDS[key],
-                        'Field has invalid value: %s (platform %s)' % (key, platform_id))
+                    self.assertTrue(
+                        platform_info[key] in REQUIRED_PLATFORM_FIELDS[key],
+                        'Field has invalid value: %s (platform %s)'
+                        % (key, platform_id))
```

Comment:
PEP8 accepts the closing function call parenthesis either on its own line or on the previous line, as we can see on line 61 and 55.

Do you have any opinions about whether you like one way or the other better?

(Not really important for this PR, just thought I'd ask)